src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,18 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
please, be attentive with [Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)